admin/src/react-components/account-edit-toolbar.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import { Toolbar, SaveButton } from "react-admin";
+import { withStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import { Dialog, DialogContent, DialogContentText, DialogActions } from "@material-ui/core";
+
+const accountEditToolbarStyles = {
+  spaceBetween: { justifyContent: "space-between" },
+  dialogActions: { padding: "0 5px" }
+};
+
+const deleteStates = {
+  confirming: "confirming",
+  deleting: "deleting",
+  succeeded: "succeeded",
+  failed: "failed"
+};
+
+export const AccountEditToolbar = withStyles(accountEditToolbarStyles)(props => {
+  const { classes, ...other } = props;
+  const [openConfirmationDialog, setOpenConfirmationDialog] = useState(false);
+  const [deleteState, setDeleteState] = useState(deleteStates.confirming);
+
+  const onDeleteAccount = async accountId => {
```

Comment:
up to you but you don't really need to pass account id here. Since it's only used once and you have access to prop.id in that function. 

 and do 
 
` const resp = await fetch(`/api/v1/accounts/${props.id}`, {`